draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1499,7 +1497,7 @@
    <iref item="no-transform (cache directive)" primary="true" />
 <t>
    The "no-transform" request directive indicates that the client is asking
-   for intermediares (whether or not they implement a cache) to avoid
+   for intermediaries (whether they implement a cache) to avoid
```

Comment:
this might be incorrect

Reply:
https://english.stackexchange.com/questions/3382/whether-or-not-vs-whether

Reply:
ok

Reply:
don't believe everything you read on the Internet

OTOH, we can delete the parenthetical.

Reply:
```suggestion
   for intermediaries to avoid
```